src/main/java/codeu/controller/AdminServlet.java

```diff
@@ -39,6 +42,23 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response) t
         return;
     }
 
+    @Override
+    protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        if (request.getParameter("deleteUsersButton") != null) {
```

Comment:
Maybe add an isAdmin check here. This seems like a very dangerous action to leave unauthenticated

Reply:
Good point, I just put an admin check around all 3 buttons.